tests/integration/basic-js-spec.js

```diff
@@ -69,12 +69,7 @@ test.describeIntegrationTest("Js Bundling:", function() {
         test.given.FileToBundle('file1.es6',
             'var odds = evens.map(v => v + 1);');
         test.given.FileToBundle('file2.es6',
-            'class Foo extends Bar {'

```

Comment:
Classes are an ES6 feature we can't support right now since they can't be transpiled in a way that works with IE8 or IE9.